gatsby/plugins/gatsby-source-cks/src/node-creation/chapters.ts

```diff
@@ -0,0 +1,90 @@
+// TODO
```

Comment:
Can remove the todo and extra whitespace between imports

Reply:
or add ```
/**
 * Creates Gatsby GraphQL nodes from downloaded data
 */
```

Reply:
Good spot, that todo doesn't need to be there any more - it's done!!